Command/Phinx/PathLocator.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace PCF\MagentoMigration\Command\Phinx;
+
+use Magento\Framework\Module\Dir\Reader;
+use Magento\Framework\App\Filesystem\DirectoryList;
+use Magento\Framework\Module\ModuleList;
+use PCF\MagentoMigration\Api\PathLocatorInterface;
+
+class PathLocator implements PathLocatorInterface
+{
+    /** @var Reader */
+    protected $dirReader;
+
+    /** @var DirectoryList */
+    protected $directoryList;
+
+    /** @var ModuleList */
+    protected $moduleList;
+
+    /**
+     * @param Reader $dirReader
+     * @param DirectoryList $directoryList
+     * @param ModuleList $moduleList
+     */
+    public function __construct(Reader $dirReader, DirectoryList $directoryList, ModuleList $moduleList)
+    {
+        $this->dirReader = $dirReader;
+        $this->directoryList = $directoryList;
+        $this->moduleList = $moduleList;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function getPhinxBinaryPath()
+    {
+        return $this->getVendorPath() . DIRECTORY_SEPARATOR . 'bin' . DIRECTORY_SEPARATOR . 'phinx';
```

Comment:
it could be great to allow get this directory from config - composer allow to move binaries to other directory.

Reply:
you are right, but I want add it in next module version.